scripts/form-utils.js

```diff
@@ -101,36 +103,21 @@ function postToBasket(email, params, url, successCallback, errorCallback) {
 
     xhr.onerror = errorCallback;
     xhr.open('POST', url, true);
-    xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+    xhr.setRequestHeader('Content-type', 'application/json');
     xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+    xhr.setRequestHeader("Access-Control-Allow-Headers", "*");
```

Comment:
Suggestion: I think you can remove this here (it's bedrock that needs to set it I believe?)

Reply:
Yep, I'm not sure that makes sense for the request. I have it in the response from bedrock. I also added the `Access-Control-Allow-Headers` list and pushing to demo2 for more testing in a bit.